src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -186,10 +187,10 @@ public static boolean isLockdown() {
      * @param defaultScheduler the hook's input value
      * @return the value returned by the hook
      */
-    public static Scheduler initComputationScheduler(Scheduler defaultScheduler) {
+    public static Scheduler initComputationScheduler(Callable<Scheduler> defaultScheduler) {
```

Comment:
This would be a change, because previously the `Scheduler` value for `RxJavaPlugins.init*` was allowed to be null, as per - `assertNull(RxJavaPlugins.initSingleScheduler(null));`. 

I will add an additional set of tests for the new behavior (something along the lines of `assemblyHookCrashes`).